src/brogue/IO.c

```diff
@@ -3920,7 +3920,7 @@ void printHighScores(boolean hiliteMostRecent) {
         }
     }
 
-    leftOffset = min(COLS - maxLength - 21 - 1, COLS/5);
+    leftOffset = min(COLS - maxLength - 23 - 1, COLS/5);
```

Comment:
Were these numbers changed to accommodate the extra two characters in the date strings?

Reply:
Yeah, line 3950 was required to offset the description two characters to the right (as the year is no 4 chars instead of 2).
Line 3923, eh, it's a guess... I tried to make several tests, but I think each time I was in the "else" part of the min() function. It made sense to me to  replace both 21 values with 23 but I may be mistaken.